zerver/lib/management.py

```diff
@@ -73,12 +73,6 @@ def add_create_user_args(self, parser: ArgumentParser) -> None:
 --password-file.""",
         )
         parser.add_argument("--password-file", help="File containing a password for the new user.")
-        parser.add_argument(
-            "--this-user-has-accepted-the-tos",
```

Comment:
Is there no value in keeping this option around for the use-case where one would want to create the new user and them **not** to be prompted to accept ToS (because it's redundant, because they've accepted it through other means)?

Reply:
Eh, on second thought no, accepting ToS is one little box to click, so not something that would be that worth skipping

Reply:
Yeah, I think in most self-hosted settings, they just won't configure ToS at at -- so the big benefit of this change is to eliminate this mandatory argument that has no actual effect for essentially all servers.